### PR TITLE
Modules set-up

### PR DESCRIPTION
**Pull request by irschad**
Terraform Modules configuration

### CODE REVIEW

File: level2/data.tf [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-kernel-*"]
+  }
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+  owners = ["137112412989"]
+}
```
Comment (geoffrey1330) [inline level2/data.tf]: could you please, add a newline at the end of file.

File: level2/loadbalancer.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module "lb" {
+  source = "../modules/lb"
+
+  env_code  = var.env_code
+  vpc_id    = data.terraform_remote_state.level1.outputs.vpc_id
+  subnet_id = data.terraform_remote_state.level1.outputs.public_subnet_id
+}
```
Comment (geoffrey1330) [inline level2/loadbalancer.tf]: could you please, add a newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level2/data.tf (hunk 1 above)
Comment (irschad) [reply level2/data.tf]: Done.

File: level2/loadbalancer.tf (hunk 2 above)
Comment (irschad) [reply level2/loadbalancer.tf]: Done.

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**